interfaces/migrate/src/main/java/migrate/interfaces/JavaLib.java

```diff
@@ -0,0 +1,34 @@
+package migrate.interfaces;
+
+
+public class JavaLib implements Lib {
```

Comment:
If you only use this class through subclasses in Scala code, you could mark it `abstract`